src/platforms/javascript/guides/capacitor/dsym.mdx

```diff
@@ -0,0 +1,7 @@
+---
+title: Uploading Debug Symbols (iOS)
+sidebar_order: 20
```

Comment:
I'm not sure where this should be in the TOC order. [@bruno-garcia](https://github.com/bruno-garcia) can advise on that. For Apple, I updated the sidebar_order to make it match where it is currently.

Reply:
Oh I see the preview was approved already

Reply:
The order LGTM